### PR TITLE
Cleaned up links and code blocks

### PR DESCRIPTION
**Pull request by Theresa-Nguyen**

### CODE REVIEW

File: ms-docs/JBoss EAP on Azure.MD [hunk 1]
```diff
@@ -72,46 +72,46 @@ Solution architecture for these templates includes:
...
+* Evaluation - JBoss EAP clusters using Azure VM or Virtual Machine Scale Set
+* Assessment - Scans applications and infrastructure
+* Infrastructure configuration - Creates workload profile
+* Deployment and testing - Deploy, migrate, and test workload
+* Post-deployment configuration - Integrates with data, monitoring, security, backup, and more
 
 ## Server configuration choice
 
-For deployment of RHEL VM, you can either choose between PAYG or BYOS. For VM image from the [Azure Marketplace](https://azuremarketplace.microsoft.com) this offer defaults to a PAYG image. Deploy a BYOS type RHEL VM if you have your own RHEL OS image. Make sure your RHSM account has BYOS entitlement via Red Hat Cloud Access before you deploy the VMs or Virtual Machine Scale Set.
+For deployment of RHEL VM, you can either choose between PAYG or BYOS. Images from the [Azure Marketplace](https://azuremarketplace.microsoft.com) defaults to PAYG. Deploy a BYOS type RHEL VM if you have your own RHEL OS image. Make sure your RHSM account has BYOS entitlement via F Cloud Access before you deploy the VMs or Virtual Machine Scale Set.
```
Comment (Suraj2093) [inline ms-docs/JBoss EAP on Azure.MD:84]: There is a letter 'F' before the word Cloud Access which I think is a typo.

File: ms-docs/JBoss EAP on Azure.MD [hunk 2]
```diff
@@ -72,46 +72,46 @@ Solution architecture for these templates includes:
...
+For a complete listing of all available startup script arguments and their purposes, use the --help argument or see the [Server Runtime Arguments on EAP 7.2] (https://access.redhat.com/documentation/en/red_hat_jboss_enterprise_application_platform/7.2/html/configuration_guide/reference_material#reference_of_switches_and_arguments_to_pass_at_server_runtime) or [Server Runtime Arguments on EAP 7.3](https://access.redhat.com/documentation/en/red_hat_jboss_enterprise_application_platform/7.3/html/configuration_guide/reference_material#reference_of_switches_and_arguments_to_pass_at_server_runtime) section.
```
Comment (Suraj2093) [inline ms-docs/JBoss EAP on Azure.MD:96]: Removed the extra space after [Server Runtime Arguments on EAP 7.2] so the hyperlink is rendered properly.

**Comment by Suraj2093** [comment]
I have made the changes in the MS Doc and added a couple of comments. Please review.

File: ms-docs/JBoss EAP on Azure.MD (hunk 1 above)
Comment (Theresa-Nguyen) [reply ms-docs/JBoss EAP on Azure.MD:84]: > There is a letter 'F' before the word Cloud Access which I think is a typo.

Please fix. Also code blocks starting in the new year needs to be this format:
'''
code
'''

**Comment by Theresa-Nguyen** [comment]
go ahead and close.

File: ms-docs/JBoss EAP on Azure.MD (hunk 1 above)
Comment (Suraj2093) [reply ms-docs/JBoss EAP on Azure.MD:84]: Noted and made these changes

**Pull request closed without merging**